packages/components/organisms/f-offers/src/components/Results.vue

```diff
@@ -0,0 +1,238 @@
+         *
+         * @param type
+         * @return {string|boolean}
+         */
+        handleCustomCardType (type) {
+            switch (type) {
+                case 'Anniversary_Card_1':
+                case 'Voucher_Card_1':
+                    return 'VoucherCard';
+                case 'Restaurant_FTC_Offer_Card':
+                    return 'FirstTimeCustomerCard';
+                case 'Promotion_Card_1':
+                case 'Promotion_Card_2':
+                    return 'PromotionCard';
+                case 'Terms_And_Conditions_Card':
+                case 'Terms_And_Conditions_Card_2':
+                    return 'TermsAndConditionsCard';
+                default:
+                    break;
+            }
+            return false;
+        },
+
+        /**
+         * Consumes the number of cards received from braze by the content cards component
+         * @param {number} cards
+         */
+        cardsReceived (cards) {
+            if (cards || this.contentCardsHaveLoaded) {
+                this.logInfo(`Content cards loaded: ${cards}`, this.getLogPayload(true));
+            }
+        },
+
+        /**
+         * Handles a successful load of the content cards component
+         */
+        hasLoaded () {
+            this.contentCardsHaveLoaded = true;
+            this.tries = 0;
+        },
+
+        /**
+         * Handles errors emitted by the content cards component
+         * @param {Error} e
+         */
+        onError (e) {
+            this.error = e;
+            this.logInfo(`Content cards error reported: "${e}"`, this.getLogPayload());
+        },
+
+        /**
+         * A simple, safe wrapper for $logger.logInfo
+         * @param {String} message
+         * @param {Object} payload
+         */
+        logInfo (message, payload) {
+            if (this.$logger && this.$logger.logInfo) {
+                this.$logger.logInfo(message, null, payload);
+            }
+        },
+
+        /**
+         * Determines the time difference since mount, and the resolution status of the cards, if given
+         * @param {boolean} resolution
+         * @returns {Object}
+         */
+        getLogPayload (resolution = undefined) {
+            return {
+                resolution,
+                timeSinceMount: Date.now() - this.mountTime
+            };
+        },
+
+        /**
+         * Generates a unique test id on a per-card basis if testId prop provided
+         * @param index
+         * @param groupIndex optional
+         */
+        testIdForItemWithIndex (index, groupIndex = null) {
+            return groupIndex !== null ?
+                this.testId && `ContentCard-${this.testId}-${index}-${groupIndex}` :
+                this.testId && `ContentCard-${this.testId}-${index}`;
+        },
+
+        /**
+         * Surfaces a message indicating a voucher code has been copied, with an optional url for an ongoing journey
+         */
+        openModal ({ url = '' }) {
```

Comment:
voucherCodeClick is not bound, so I don't think this will be called.

Reply:
have bound this now so that if voucher content card is used then it fires